htdocs/edittags.bml

```diff
@@ -103,8 +103,11 @@ body<=
     $ret .= "<script type='text/javascript'> var cur_taglist = '$logtagstr'; </script>";
 
     $ret .= '<table summary="" class="edittbl" cellpadding="0" cellspacing="0" width="50%">';
+    if ( $u->is_community ) {
+        $ret .= "<tr><td class='l'>$ML{'.poster'}</td><td>" . $ent->poster->ljuser_display . "</td></tr>"
+        };
```

Comment:
This works fine now, thanks! I need to do a little style nit-pick though: the semicolon should be immediately after the `</td></tr>`, and the closing `}` should be on a line by itself, indented at the same level as the if statement.